deployments/mdpresentation-viewer.yml

```diff
@@ -0,0 +1,16 @@
+
+---
+services:
+  ocis:
+    depends_on:
+      mdpresentation-viewer:
+        condition: service_completed_successfully
+
+  mdpresentation-viewer:
+    image: jankaritech/mdpresentation-viewer:2.0.0
```

Comment:
if you use the `latest` tag instead of the current release version, you don't have to update this file with each release :)

Reply:
Thanks. I will update this.
Currently, I have released 2.0.0 version but latest tag will be build by github action . I will look over this.